drivers/census_tracking/app/models/census_tracking/worksheet.rb

```diff
@@ -21,7 +21,7 @@ def projects
         viewable_by(@filter.user).
         where(id: @filter.effective_project_ids).
         map do |project|
-          [HUD.project_type(project.computed_project_type), project.organization.name, project.safe_project_name, project.id]
+          [HUD.project_type(project.computed_project_type) || 'Unknown Project Type', project.organization.name, "#{project.safe_project_name} (#{project.ProjectID})", project.id]
```

Comment:
Since this doesn't run in a background job, can we use the appropriate project name instead?  Maybe: `Project.name_and_type(include_confidential_names: current_user.can_view_confidential_enrollment_details?)`

Reply:
Not perfect, so it might be worth adding a method to return the name if you can see it and the safe name and id if you can't.  I'm on the fence about ProjectID vs id, id would be more useful within the warehouse since you can't search by ProjectID.